acceptance/bundle/variables/file-defaults/test.toml

```diff
@@ -0,0 +1,8 @@
+# Fix for windows
+[[Repls]]
+Old = '\$TMPDIR\\.databricks\\bundle\\wrong_file_structure\\variable-overrides.json'
+New = '$TMPDIR/.databricks/bundle/wrong_file_structure/variable-overrides.json'
```

Comment:
The `$` is interpreted as a reference to match. You can indicate a `$` literal with `$$`:

https://pkg.go.dev/regexp#Regexp.Expand